diksha/diksha.init.js

```diff
@@ -117,6 +117,9 @@ let initialize = () => {
         return createFolderIfNotExists(dikshaData.media_root);
     }).then(value => {
         console.log("Created " + dikshaData.media_root);
+        return createFolderIfNotExists(`${dikshaData.media_root}/.stfolder`);
```

Comment:
Why do you need to add .stfolder ? 

How was it working earlier, without this folder?

Reply:
I did not test it on a fresh build so the .stfolder was in `/home/admin/diksha` there from before. That's why I missed it.